### PR TITLE
dev-python/frozendict: add 2.3.4 & fixed test phase

### PR DESCRIPTION
**Pull request by arkamar**
Straight forward version bump, ~nothing has changed from ebuild point of view. Tests still fail~ with fixed test phase.

### CODE REVIEW

File: dev-python/frozendict/frozendict-2.3.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A simple immutable mapping for python"
+HOMEPAGE="
+	https://github.com/Marco-Sulla/python-frozendict
+	https://pypi.python.org/pypi/frozendict
```
Comment (mgorny) [inline dev-python/frozendict/frozendict-2.3.4.ebuild:14]: Missing slash. `pkgcheck scan --net` is your friend.

File: dev-python/frozendict/frozendict-2.3.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+"
+SRC_URI="https://github.com/Marco-Sulla/python-frozendict/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
```
Comment (mgorny) [inline dev-python/frozendict/frozendict-2.3.4.ebuild:16]: Wrap, please.

File: dev-python/frozendict/frozendict-2.3.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (mgorny) [inline dev-python/frozendict/frozendict-2.3.4.ebuild:17]: No empty line here.

File: dev-python/frozendict/frozendict-2.3.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+S="${WORKDIR}/python-${P}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64"
+
+# few tests fail with python3_9 and one with python3_10
+RESTRICT="test"
```
Comment (mgorny) [inline dev-python/frozendict/frozendict-2.3.4.ebuild:25]: What did you do to get them fixed?

**Review by mgorny — CHANGES_REQUESTED** [review]

File: dev-python/frozendict/frozendict-2.3.4.ebuild (hunk 1 above)
Comment (arkamar) [reply dev-python/frozendict/frozendict-2.3.4.ebuild]: Thanks! I did not know this one.

Comment (arkamar) [reply dev-python/frozendict/frozendict-2.3.4.ebuild]: I mean the `--net` parameter :)

File: dev-python/frozendict/frozendict-2.3.4.ebuild (hunk 4 above)
Comment (arkamar) [reply dev-python/frozendict/frozendict-2.3.4.ebuild]: Not enough. Anyway, I looked to it now and fixed it. I did the fix in `2.3.2` as well.

**Review by mgorny — APPROVED** [review]
Thank you, great work!

**Comment by arkamar** [comment]
Thanks!

**Pull request closed without merging**